docs/ui/auth/fragments/vue/overriding-sign-up-submit.md

```diff
@@ -0,0 +1,31 @@
+_App.vue_
+
+```html
+<template>
+    <amplify-authenticator>
+        <amplify-sign-up
+            slot="sign-up"
+            :handleSignUp.prop="handleSignUp"
+        ></amplify-sign-up>
+    </amplify-authenticator>
+</template>
+<script>
+import { Auth } from 'aws-amplify';
+
+export default {
+	name: 'Authenticator',
+	methods: {
+		handleSignUp(data) {
```

Comment:
Where is `data` used?

Reply:
Ah, this should be `formData`! Thanks for the catch.
```suggestion
		handleSignUp(formData) {
```